### ISSUE TITLE
Test for DH Parameter support

### ISSUE THREAD

**Issue by fishouttah20** [jmhodges/howsmyssl#47]
Given the recent Logjam announcement it would be helpful to understand the maximum DH Parameter size supported by each tested client. The current recommendation is to use 2048 bit DH parameters but some clients, most notably Java 6/7, only support 1024 bit DH.

**Comment by rugk** [comment]
Indeed. More information here: https://www.eff.org/deeplinks/2015/10/how-to-protect-yourself-from-nsa-attacks-1024-bit-DH
